packages/headless-components/events/src/react/core/Tag.tsx

```diff
@@ -0,0 +1,64 @@
+import { useService, WixServices } from '@wix/services-manager-react';
+import { createServicesMap } from '@wix/services-manager';
+import React from 'react';
+import {
+  TagService,
+  TagServiceDefinition,
+  type TagServiceConfig,
+} from '../../services/tag-service.js';
+
+export interface RootProps {
+  /** Child components that will have access to the tag service */
```

Comment:
let's name to `ScheduleItemTag` (components, service, etc) - `Tag` is too generic